File: 2298-count-integers-with-even-digit-sum/count-integers-with-even-digit-sum.py
class Solution:
    def countEven(self, num: int) -> int:
        def digitSum(x):
            sumOfDigit = 0
            while x > 0:
                sumOfDigit += x % 10
                x //= 10
            return sumOfDigit
        
        count = 0
        for i in range(1, num + 1):
            if digitSum(i) % 2 == 0:
                count += 1
        return count

File: 2298-count-integers-with-even-digit-sum/count-integers-with-even-digit-sum.java
class Solution {
    public int countEven(int num) {
        int count = 0;
        for (int i = 1; i <= num; i++){
            if (digitSum(i) % 2 == 0) count++;
        }
        return count;
    }

    public int digitSum(int x){
        int sumOfDigit = 0;
        while (x > 0){
            sumOfDigit += x % 10;
            x /= 10;
        }
        return sumOfDigit;
    }
}